classes/core.php

```diff
@@ -1446,8 +1450,17 @@ static public function run_calculation($value, $field, $form)
 				__('Calculation is invalid (division by zero)', 'caldera-forms'));
 		}
 
-		$total_function = create_function(null, 'return ' . $formula . ';');
-		$total = $total_function();
+		/**
+		 *  Set result out of formula Parser 
+		 *
+		 * @since 1.8.5
+		 * 
+		 * @param $formula 
+		 * @param $precision non required, Number of digits after the decimal point
+		 */
+		$parser = new FormulaParser($formula);
```

Comment:
@New0 Let's revert this until we have all of the tests in place. Once they pass as-is, then we swap out the parser.